docs/configuring-metabase/localization.md

```diff
@@ -6,6 +6,54 @@ redirect_from:
 
 # Languages and localization
 
+The **Localization** settings allow you to set global defaults for your Metabase instance. You can find **Localization** under **Admin settings** > **Settings**.
+
+## Instance language
```

Comment:
```suggestion
## Default language
```

I don't imagine many people will be searching for "instance language".

Reply:
Oh, this just matches the in-product name (in case people are looking at the docs and the interface at the same time). I'll add an "aka" in the description.